sdk/core/core-http/src/policies/requestPolicy.ts

```diff
@@ -13,16 +13,39 @@ export type RequestPolicyFactory = {
   create(nextPolicy: RequestPolicy, options: RequestPolicyOptionsLike): RequestPolicy;
 };
 
+/**
+ * The underlying structure of a request policy.
+ */
 export interface RequestPolicy {
+  /**
+   * In principle, a request policy is only a structure with a method that sends an HTTP request.
+   * @param httpRequest - Object representing the outgoing HTTP request.
+   */
   sendRequest(httpRequest: WebResourceLike): Promise<HttpOperationResponse>;
 }
 
+/**
+ * The base class that defines all request policies.
+ */
 export abstract class BaseRequestPolicy implements RequestPolicy {
+  /**
+   * The main method to implement that manipulates a request/response.
+   */
   protected constructor(
+    /**
+     * The next policy in the pipeline.Each policy is responsible for executing the next one.
+     */
     readonly _nextPolicy: RequestPolicy,
+    /**
+     * The options that can be passed to a given request policy.At a minimum should contain logging properties.
```

Comment:
We don't really use the logging properties anymore, since we have log policy. I think they can safely be not discussed.

```suggestion
     * The options that can be passed to a given request policy.
```